lib/less/tree/ruleset.js

```diff
@@ -182,15 +182,15 @@ Ruleset.prototype.evalImports = function(context) {
     }
 };
 Ruleset.prototype.makeImportant = function() {
-    return new Ruleset(this.selectors, this.rules.map(function (r) {
-                if (r.makeImportant) {
-                    return r.makeImportant();
-                } else {
-                    return r;
-                }
-            }), this.strictImports);
-};
-Ruleset.prototype.matchArgs = function (args) {
+    this.rules = this.rules.map(function (r) {
```

Comment:
I was initially concerned, but this is only used from one place which has already copied the Ruleset.
Its a good change - but for my own curiousity, what is wrong with the new Ruleset that is causing un-eval'd rules?